bibcheck/rules.cfg

```diff
@@ -24,3 +24,12 @@ filter_collection = HEP
 
 [check_earliest_date]
 check = earliest_date
+
```

Comment:
@kaplun Is it safe to add this to the rules.cfg or will it go haywire on production?